lightning/src/ln/payment_tests.rs

```diff
@@ -526,7 +528,24 @@ fn test_mpp_keysend() {
 
 	let ev = remove_first_msg_event_to_node(&node_c_id, &mut events);
 	pass_along_path(&nodes[0], route[1], recv_value, hash, payment_secret, ev, true, preimage);
-	claim_payment_along_route(ClaimAlongRouteArgs::new(&nodes[0], route, preimage.unwrap()));
+
+	#[cfg(feature = "std")]
+	thread::sleep(Duration::from_millis(200));
+
+	let (_, path_events) =
+		claim_payment_along_route(ClaimAlongRouteArgs::new(&nodes[0], route, preimage.unwrap()));
+
+	// If we've delayed, check that this is reflected in the hold times.
```

Comment:
Can we also check that the hold times are decreasing along the path?

Reply:
Added.